docs/tools/offline/cli-device-management.md

```diff
@@ -2,6 +2,12 @@
 
 Arm Mbed OS allows you to update your device firmware, enabled by the Pelion IoT platform. Mbed CLI includes features to prepare and ship updates for devices managed through the [Device Management Portal](https://cloud.mbed.com/docs/current/introduction/index.html). Mbed CLI provides the subcommand `mbed device-management` to manage devices (`mbed dev-mgmt` and `mbed dm` are also available as shorter aliases). The remainder of this document uses the `mbed dm` alias for all device management subcommands. This document explains the steps to enable and use Pelion Device Management with a project.
```

Comment:
Pelion Device Management IoT platform, when mentioned for the first time. Then just Device Management (no Pelion).